test/tap/tests/pgsql-user-sync-t.py

```
@@ -0,0 +1,270 @@
+    with admin_connection() as connection:
+        with connection.cursor() as cursor:
+            cursor.execute(query, params)
+
+
+def admin_row(username, runtime=False):
+    table = "runtime_pgsql_users" if runtime else "pgsql_users"
+    with admin_connection() as connection:
+        with connection.cursor() as cursor:
+            cursor.execute(
+                f"SELECT username,password,active,default_hostgroup,attributes "
+                f"FROM {table} WHERE username=%s",
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)